app/js/shortID.js

```diff
@@ -0,0 +1,4 @@
+module.exports = function generate(str) {
+	return (
+		str.concat(String(Math.floor(Math.random() * ((9999999 - 9) + 9)))));
```

Comment:
hehe, I was a bit unsure about using this. This is going to generate a unique key when users create their own recipe to ensure the mapped recipes have their own unique key value.